src/main/java/filter/expression/Qualifier.java

```diff
@@ -28,10 +28,10 @@
     public static final Qualifier FALSE = new Qualifier("false", "");
 
     public static final List<String> KEYWORDS = Collections.unmodifiableList(Arrays.asList(
-        "assignee", "author", "body", "closed", "comments", "created", "creator",
+        "assignee", "as", "author", "au", "body", "closed", "comments", "created", "creator",
         "date", "desc", "description", "false", "has", "id", "in", "involves",
-        "is", "issue", "keyword", "label", "labels", "merged", "milestone", "milestones",
-        "no", "nonSelfUpdate", "open", "pr", "pullrequest", "read", REPO, SORT, "state", "status",
+        "is", "issue", "keyword", "label", "labels", "merged", "milestone", "m", "milestones",
+        "no", "nonSelfUpdate", "open", "pr", "pullrequest", "read", REPO, SORT, "state", "status", "s",
         "title", "type", "unmerged", "unread", UPDATED, "user"
     ));
```

Comment:
Yes, we should use an enum for keywords.

About this specific change: this keyword list is used for autocompletion, so I don't think we need to have the abbreviations here.